csi.proto

```diff
@@ -1296,6 +1357,31 @@ message VolumeUsage {
   // Units by which values are measured. This field is REQUIRED.
   Unit unit = 4;
 }
+
+// VolumeCondition represents the current health condition of a volume
+message VolumeCondition {
+  option (alpha_message) = true;
+
+  // Status represents the current status of a condition
+  message Status {
+    enum Type {
+      CONDITION_UNKNOWN = 0;
+      CONDITION_TRUE = 1;
+      CONDITION_FALSE = 2;
+    }
+    Type type = 1;
+  }
+
+  oneof type {
```

Comment:
Any reason for the oneof here?

Reply:
When we add more types in the future, only 1 should be specified.
I dropped "oneof" now.